kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/wifi/WifiChannel.java

```diff
@@ -0,0 +1,69 @@
+package org.eclipse.kura.net.wifi;
```

Comment:
A copyright header is required for all the code contributions

Reply:
OK.